lib/LANraragi/Plugin/Metadata/nHentai.pm

```diff
@@ -220,12 +216,6 @@ sub get_tags_from_NH {
     my %hashdata = ( tags => "" );
 
     my $html = get_html_from_NH( $gID, $ua );
-
-    # If the string starts with "error", we couldn't retrieve data from NH.
-    if ( $html =~ /^error/ ) {
```

Comment:
Why was this block removed?

Reply:
Now it fails inside `get_html_from_NH(...)` and dies.

Reply:
right I hadn't read this fully - thanks 👍